tests/test_diff_rrules.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+
+from dateutil.rrule import MONTHLY, rrule
```

Comment:
`dateutil` is not a dependency of DeepDiff. So we can have it imported just for this test. It needs to be imported inside the test function then, and `dateutil` needs to be added to the `requirements-dev.txt` so Github actions install them before running the tests.

Reply:
I had it somehow in my environment, but you are right, it needs to be added.